coderd/notifications_test.go

```diff
@@ -115,7 +115,7 @@ func TestNotificationPreferences(t *testing.T) {
 	t.Run("Initial state", func(t *testing.T) {
 		t.Parallel()
 
-		ctx := testutil.Context(t, testutil.WaitLong)
+		ctx := testutil.Context(t, testutil.WaitSuperLong)
```

Comment:
What do we end up waiting for? Super long is 1m IIRC, most tests don't need that much.

Reply:
Might as well wait a while, especially since we're reaching out to a real database for all tests now.